quickstart.md

```diff
@@ -45,12 +45,12 @@ This script requires `ansible-playbook`, which you can install on CentOS/RHEL us
 `dnf install ansible-core`, and the Ansible `docker_container` module, which you can
 get using `ansible-galaxy collection install community.docker`.
 
-> **Note** You can also use a Kind or Minikube cluster with containerd runtime to try out the CoCo stack
+> ****************************************** **Note** You can also use a Kind or Minikube cluster with containerd runtime to try out the CoCo stack
```

Comment:
why so many ****'s?

Reply:
Sorry I had marked that line as a note to myself, will have that updated.